Firestore/core/src/firebase/firestore/util/comparison.cc

```diff
@@ -19,11 +19,10 @@
 #include <cmath>
 #include <limits>
 
-using std::isnan;
```

Comment:
What's the reason for this change?

Reply:
Possibly this: http://go/totw/119#scope-of-the-alias

Though it seems otherwise unrelated to this PR, so maybe there's a different reason?

Reply:
The reason is already in the description -- resolve conflict. To be precise, this error *target of using declaration conflicts with declaration already in scope*

Reply:
Yes, I'm not sure where the unqualified `isnan` is coming from, does something include `<math.h>`? Likely not worth digging into, though.